molecule/resources/example_configs/logging_custom_config.conf

```diff
@@ -0,0 +1,62 @@
+# Sample custom config
```

Comment:
I  think it would be helpful if you described where you got this conf file from. Is it a copy of something else or was it all written by hand or what? Same with the other config files.